Added randomcase method and some CSS

KingShimkus opened the pull request:
So I added the `randomcase` method which generates a 50/50 chance using the global `Math` variable, etc. So the returned string letters are randomly changed to upper or lowercase. I also updated some of the CSS, I don't know how you will feel about it. I personally think that it looks better with my additions/deletions. Also added `randomcase` to the tests. But it is like the `randomize` method in that it still can fail. I hope you enjoy my code. Love the repo. And hope to contribute more. :)

Commit:
Added randomcase method

I added the randomcase method which randomly set letters upper and lowercase.

Commit:
Changed the way it looked and feels

Honestly, I think you will like my changes here, I really made it look nicer and changed colors. (in my opinion) :)

Commit:
Added test for randomcase method

I know you that it is still possible for randomcase method to still equal the origanal string, kind of like the randomize method. How should we go about getting around this?

KingShimkus commented:
Oh yeah, I also didn't want to update/edit the `README.md` until I get your approval. I know you said in `CONTRIBUTING.md` that you wanted us to update the README to but I don't know... :+1:

salexzee commented:
Love your code. I have no issue with the CSS changes on the test page. And yes please update the README. Thanks for PR and the compliment on the repo.

salexzee commented on src/Easify.js:
```diff
@@ -205,6 +205,35 @@
+        var newString = '';
+        for (var i = 0; i < arr.length; i++) {
+          // Generate a 50/50 chance
+          if(Math.round(Math.random() * 1) == 1){
```
Please make sure to use strict equality (===) instead of non strict equality (==).

Commit:
Added  strict equality

Commit:
Added documentation

KingShimkus commented:
Okay, I added another `=` sign and added documentation to the `README`. :)

Commit:
Added code highlighting and hr

This code 'highlights' the method name and makes it easier to read.

Commit:
Updated id names

I changed the id names so it will be compatible with the CSS and Javascript.

KingShimkus commented:
Hey, so I updated a lot in the `tests` directory because I saw that the css was not really compatible with the right HTML id names and such, and also added highlighting for the method names in Javascript. I hope you like what I did.

Commit:
Update tests.js

Commit:
OOps

salexzee commented:
I'll probably be changing `test-styles.css` to `tests.css` in the next version, but for now it needs to keep the original filename to prevent merge conflicts when I merge the rest of the changes.

Commit:
Changed names

Commit:
Changed ex to $E

KingShimkus commented:
Fixed things.

Commit:
Update index.html

KingShimkus commented:
Okay, I think I fixed everything. Next pull request won't be as sloppy, I swear. :)
